src/backup/abstract_backup_predicate.py

```diff
@@ -12,13 +13,15 @@ def test(self, big_query_table_metadata, table_entity):
     def _is_possible_to_copy_table(big_query_table_metadata):
         if not big_query_table_metadata.table_exists():
             logging.info('Table not found (404)')
-            return False
+            return False, "Table not found"
         if not big_query_table_metadata.is_schema_defined():
             logging.info('This table is without schema')
-            return False
+            return False, "This table is without schema"
         if big_query_table_metadata.is_external_or_view_type():
-            return False
+            logging.info('This table is external or view type')
+            return False, "This table is external or view type"
         if big_query_table_metadata.is_empty():
             logging.info('This table is empty')
+            return False, "This table is empty"
```

Comment:
It's tricky, but there are cases, when even if table is empty, we want to copy it. 

Not returning anything in previous implementation was intended.